cli/ceph/fs/subvolume/sub_volume.py

```diff
@@ -69,16 +68,15 @@ def rm(self, volume, subvolume, group=None, force=False):
             return out[0].strip()
         return out
 
-    def ls(self, volume, group=None):
+    def ls(self, volume, **kwargs):
```

Comment:
I think we may have to cross check with other fg's before changing this. The reason being, its been used by other scripts (for eg. smb)

```        for smb_subvol in smb_subvols:
            CephAdm(installer).ceph.fs.sub_volume.rm(
                cephfs_vol, smb_subvol, smb_subvol_group
            )```
            
            the group is specifically mentioned here and changing will impact these exisitng usages. 
            Please have a look. Let me know if this is already handled